backend/src/exceptions/exception.fillter.ts

```diff
@@ -0,0 +1,28 @@
+import { Catch, ExceptionFilter, ArgumentsHost, HttpException, HttpStatus } from '@nestjs/common';
+import { Response } from 'express';
+import { ValidationException } from './validation.exception';
+
+@Catch(HttpException)
+export class DuplicateEmailExceptionFilter implements ExceptionFilter {
+    catch(exception: HttpException, host: ArgumentsHost) {
+        const ctx = host.switchToHttp();
+        const response = ctx.getResponse<Response>();
+
+        if (exception.message.includes('Duplicate entry')) {
+            this.setResposne(response, HttpStatus.CONFLICT, '이메일 중복');
+            return;
+        }
+        if (exception instanceof ValidationException) {
+            this.setResposne(response, exception.getStatus(), exception.getMessage());
+            return;
+        }
+        this.setResposne(response, HttpStatus.CONFLICT, '서버 내부 에러');
```

Comment:
이 부분은 서버 내부 에러로 처리하시려고 하신거면 CONFLICT가 아니어야 할 것 같아요!
```suggestion
        this.setResposne(response, HttpStatus.INTERNAL_SERVER_ERROR, '서버 내부 에러');
```